ui/admin/tests/acceptance/groups/selectors.js

```diff
@@ -5,4 +5,13 @@
 
 export const MANAGE_DROPDOWN =
   '[data-test-manage-group-dropdown] button:first-child';
-export const DELETE_ACTION = '[data-test-manage-group-dropdown] ul li button';
+export const MANAGE_DROPDOWN_DELETE_GROUP =
+  '[data-test-manage-group-dropdown] ul li button';
+
+// Members
+export const TABLE_MEMBER_ACTION_DROPDOWN =
+  '[data-test-group-member-action] button:first-child';
+export const MANAGE_DROPDOWN_ADD_MEMBER =
+  '[data-test-manage-group-dropdown] ul li a';
+export const TABLE_ACTION_DROPDOWN_DELETE_MEMBER =
```

Comment:
I think I made a mess on my previou explanation about DELETE_MEMBR_ACTION. But since this selector is an option of the `TABLE_MEMBER_ACTION_DROPDOWN`, can we call this one `TABLE_MEMBER_ACTION_DROPDOWN_DELETE_MEMBER` ?